math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -3242,6 +3242,203 @@ RVec<typename RVec<T>::size_type> Enumerate(const RVec<T> &v)
    return ret;
 }
 
+/**
+ * @brief Produce RVec with N evenly-spaced entries from start to end inclusive.
+ *
+ * This function generates a vector of evenly spaced values, starting at @p start and ending at @p end,
+ * with exactly @p n elements. The spacing is computed as
+ * \f$\text{step} = \frac{\text{end} - \text{start}}{n-1}\f$, so that the vector includes both endpoints.
```

Comment:
Throughout the ROOT codebase, the Doxygen token used as sequence beginner is `\` rather than `@`. See for example the docs of `Construct` from above: `\begin`, `\tparam`, `\p` etc. Please move all the `@` instances to `\` instead.